### PR TITLE
chore: update Spinner component styles

### PR DESCRIPTION
**Pull request by thekidnamedkd**
## Description

Updates "neutral" style to new monochrome look, adds new "primary_inverted" variant

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [x] I have tested my code on the test network.

### CODE REVIEW

File: src/core/components/spinner/spinner.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type { ResponsiveAttribute, ResponsiveAttributeClassMap } from '../../typ
 import { responsiveUtils } from '../../utils';
 
 export type SpinnerSize = 'sm' | 'md' | 'lg' | 'xl';
-export type SpinnerVariant = 'neutral' | 'primary' | 'success' | 'warning' | 'critical';
+export type SpinnerVariant = 'neutral' | 'primary' | 'primary_inverse' | 'success' | 'warning' | 'critical';
```
Comment (Fabricevladimir) [inline src/core/components/spinner/spinner.tsx:7]: Are you certain that the inverse is only for the primary?

**Review by Fabricevladimir — APPROVED** [review]
LGTM! Don't forget to rebase and fix the changelog conflicts

**Review by Fabricevladimir — COMMENTED** [review]
Need to sync with design

**Review by Fabricevladimir — CHANGES_REQUESTED** [review]
Need to sync with design

**Comment by cgero-eth** [comment]
Closing as updates have been merged into https://github.com/aragon/ods/pull/194

**Pull request closed without merging**